examples/kafka_spark_demo/feature_repo/streaming_features_tutorial.ipynb

```diff
@@ -0,0 +1,841 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Building streaming features with Feast, Spark, Kafka"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Overview\n",
+    "\n",
+    "This notebook builds off of the first [module](https://github.com/feast-dev/feast-workshop/tree/main/module_1) from the Feast workshop to explore how data scientists and engineers can build and register streaming features in Feast. Note that the workshop module is not a prerequisite for this tutorial, but is recommended if you are not already familiar with the Push API in Feast.\n",
+    "\n",
+    "Feast currently does not launch or manage any ingestion or stream transformation jobs. If you want to use streaming features, you will be responsible for ingesting data from your streams, applying any desired transformations, and then writing that data to the online store. Additionally, if you want to use that data for training models, you will also need to persist it to some batch data source, which must then be registered with Feast. (maybe should note unified push API here?)\n",
+    "\n",
+    "Although Feast does not launch and manage streaming jobs, it can still greatly improve the user workflows around streaming features. Specifically, data scientists and engineers can register streaming features with transformations. Then during the ingestion and transformation pipeline, the jobs can simply pull the transformation associated with the stream feature view and execute it; this ensures that transformations are properly versioned within Feast, giving the entire team more visibility.\n",
```

Comment:
any thoughts on how users think about batch transformations as they relate to this?

one thing that I think is true, but isn't intuitive to me is that the "batch source" of the stream source has transformed feature values, instead of the raw values that are post processed by the streamfeatureview udf.

Reply:
we've already had some discussions offline about this, here's my current thinking: for right now the batch source must have the transformed features, in the future we will support both options (either batch source has transformed features, or non-transformed in which case the sfv udf is applied at historical retrieval time)

one awkward thing we'll have to think about is how this interacts with unified push API - if a team uses unified push API to push transformed features to offline and online stores, does this preclude them from using previous batch source with non-transformed features?